lib/rules/no-multi-spaces.js

```diff
@@ -99,3 +99,21 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.schema = [
+    {
+        "type": "object",
+        "properties": {
+            "exceptions": {
+                "type": "object",
+                "patternProperties": {
+                    "^([A-Z][a-z]*)+$": {
```

Comment:
Don't think so. I'm looking to match ESTree node types, which, so far, are PascalCased. I think that's what this does.